### DISCUSSION TITLE
trigger systemVerilog event with cocotb.

### DISCUSSION THREAD

**Discussion by M0stafaRady** [cocotb/cocotb#3766] [Q&A] [answered]
Hello, 
I'm using a systemVerilog vip that has an event for reset. 
`event reset;` 

I need to trigger this event using the testbench. In systemVerilog testbench I could use `-> vip.reset` to trigger the event. what is the equivalent for this in cocotb ?

**Comment by ktbarrett** [comment, marked as answer]
This is not directly possible from cocotb currently. Instead what could be done is to make a "driver" in SV that waits for a change on a "start" signal and then does `-> vip.reset`. Then from cocotb you can simply toggle the driver's "start" signal.

**Reply by M0stafaRady** [reply]
Thanks that would work for me.
